qa/L0_infer/infer_test.py

```diff
@@ -56,8 +57,14 @@
 
 class InferTest(tu.TestResultCollector):
 
-    def _full_exact(self, input_dtype, output0_dtype, output1_dtype,
-                    output0_raw, output1_raw, swap):
+    def _full_exact(self,
+                    input_dtype,
+                    output0_dtype,
+                    output1_dtype,
+                    output0_raw,
+                    output1_raw,
+                    swap,
+                    network_timeout=60.0):
```

Comment:
Why are we adding the network timeout for this test?

Reply:
We only increase `network_timeout `for TensorFlow models for valgrind test. The default value of  `network_timeout ` is 60.0. (https://github.com/triton-inference-server/client/blob/main/src/python/library/tritonclient/http/__init__.py#L155)